src/backupNg/backupNg.spec.js

```diff
@@ -30,5 +30,38 @@ describe("backupNg", () => {
       expect(omit(backupNg, "id", "userId")).toMatchSnapshot();
       expect(backupNg.userId).toBe(xo._user.id);
     });
+
+    it("creates a new backup job with schedules", async () => {
+      let scheduleId = "scheduleTempId";
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
+          ...defaultBackupNg.settings,
+          [scheduleId]: { snapshotRetention: 1 },
+        },
+      });
+
+      const backupNgJob = await xo.call("backupNg.getJob", { id: jobId });
+      for (const key in backupNgJob.settings) {
+        if (key !== "") {
+          expect(backupNgJob.settings[key]).toEqual({ snapshotRetention: 1 });
+          scheduleId = key;
```

Comment:
@badrAZ , `let` -> `const` even if `scheduleId` may be modified ?

Reply:
The first schedule id var is a temporary const defined by we to link the schedule with the job setting. The second is a variable used to get the new schedule id.